tpchgen-cli/src/main.rs

```diff
@@ -105,6 +105,11 @@ struct Cli {
     /// Verbose output (default: false)
     #[arg(short, long, default_value_t = false)]
     verbose: bool,
+
+    /// Write the output to stdout instead of a file, this is only supported
+    /// for the tbl formats.
+    #[arg(long, default_value_t = false)]
+    stdout: bool,
```

Comment:
It might also be good to error or warn if the user has specified `--stdout` and `--format=parquet` (like a warning it will be ignored, for example)